### PR TITLE
Active tracing changes

### PR DESCRIPTION
**Pull request by shukrash**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-synthetics-canary/template.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ Resources:
     Type: AWS::Serverless::Function
     Properties:
       Handler: com.amazon.synthetics.canary.HandlerWrapper::testEntrypoint
+      MemorySize: 256
```
Comment (dschinde) [inline aws-synthetics-canary/template.yml:21]: This shouldn't be commited.

```suggestion
```

File: aws-synthetics-canary/src/main/java/com/amazon/synthetics/canary/UpdateHandler.java [hunk 2]
```diff
@@ -148,6 +150,11 @@
                 logger.log("Updating memory");
                 memoryInMB = model.getRunConfig().getMemoryInMB();
             }
+            
+            if(model.getRunConfig().getActiveTracing() != null && !Objects.equals(activeTracing, model.getRunConfig().getActiveTracing())) {
```
Comment (dschinde) [inline aws-synthetics-canary/src/main/java/com/amazon/synthetics/canary/UpdateHandler.java:154]: Formatting:

```suggestion
            if (model.getRunConfig().getActiveTracing() != null && !Objects.equals(activeTracing, model.getRunConfig().getActiveTracing())) {
```

**Review by dschinde — CHANGES_REQUESTED** [review]

File: aws-synthetics-canary/src/main/java/com/amazon/synthetics/canary/UpdateHandler.java (hunk 2 above)
Comment (shukrash) [reply aws-synthetics-canary/src/main/java/com/amazon/synthetics/canary/UpdateHandler.java]: Done!

File: aws-synthetics-canary/template.yml (hunk 1 above)
Comment (shukrash) [reply aws-synthetics-canary/template.yml]: Done!

**Review by dschinde — APPROVED** [review]

**Pull request merged**